nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -693,7 +693,8 @@ def exec(self):
         self.output_config.fullExplanation = True  # assign true for this query - it is always ok to compare its results
         self.output_config.configName = os.path.basename(self.config.name) if self.config.name.startswith('./') else \
             self.config.name
-        peers_to_compare = self.config.peer_container.get_all_peers_group(include_dns_entries=True)
+        peers_to_compare = \
+            self.config.peer_container.get_all_peers_group(include_dns_entries=self.configs_contain_istio_layer())
```

Comment:
why not using `get_all_peers_group(include_dns_entries=True)`?
If there is no `istio` layer then such peers will not be added anyway.
Same question for 2-config queries.

Reply:
changed `get_all_peers_group(include_dns_entries=True)` for all queries.
In next PRs , we are planning to create policy elements from the `ServiceEntry ` objects besides to `DNSEntry` peers.
I think this will resolve the "implicit" existence of `Istio layer` when `ServiceEntries ` are provided